platform/viewer/pluginConfig.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Packages are things that are available either as package modules,
+ * link modules, or umd modules.  To distinguish them:
+ * 1. If the <viewer>/node_modules/<packageName> exists, it is a package module
+ * 2. If the directory attribute exists, it is a UMD module
+ * 3. Otherwise it is a link module
+ *
+ * Package and link modules are available via the "importer" on pluginImport
+ * umd modules, if they export "public/umd/<packageName>/index.umd", are
+ * available via the umdImport module.
+ * Both are exposed via the umdImport module.
+ *
+ * Packages can be set to hot-load by setting hot to true.  This means
+ * it will load at pluginImports load time.  This happens before the
+ * defaultExtensions is defined, so it then becomes possible to define
+ * custom default extension handling.
+ *
+ * The packages are grouped into three types:
+ *   extensions - this is the default extensions list, always loaded if present
+ *   modes - the default modes list, only loaded if the config doesn't define modes
+ *   packages - other packages that are dependencies, including OHIF itself.
+ *     Define a package from OHIF if you want to use it as an external inside
+ *     a umd module.
+ */
+module.exports = {
+  /**
+   * Specifies the set of directories for importing UMD modules, as well
+   * as for the yarn cli interface to allow checkout from git and linking.
+   * All of these references are optional, used only if specified.
+   */
+  directories: {
+    // Defines the src directory
+    src: {
+      relativeTo: 'OHIF',
+      directory: '..',
+    },
+
+    // Defines an optional set of modes and extensions for checkout.
+    OHIFExtensionsAndModes: {
+      relativeTo: 'src',
+      directory: 'OHIFExtensionsAndModes',
+      git: 'https://github.com/RadicalImaging/OHIFExtensionsAndModes.git',
+    },
+
+    'config-point': {
+      relativeTo: 'src',
+      directory: 'config-point',
+      git: 'https://github.com/OHIF/config-point.git',
+    },
+  },
+
+  /**
+   * This is the default list of extensions loaded on startup, before the
+   * extensions listed in the config.
+   */
+  extensions: [
+    '@ohif/extension-cornerstone',
+    '@ohif/extension-measurement-tracking',
+    '@ohif/extension-cornerstone-dicom-sr',
+    '@ohif/extension-default',
+    '@ohif/extension-dicom-pdf',
+    '@ohif/extension-dicom-video',
+  ],
+
+  /**
+   * This is the default list of modes, loaded on startup.
+   * The app config can over-ride this list by specifying an alternate set of modes.
+   */
+  modes: ['@ohif/mode-longitudinal'],
+
+  /**
+   * Packages needed to be known about, but not extensions/modes
+   */
+  packages: [
+    '@ohif/core',
+    '@ohif/ui',
+    '@ohif/extension-tmtv',
+    '@ohif/mode-tmtv',
+    'react',
+
+    // All the remaining packages are optional, included only if the
```

Comment:
The definitions are all optional, but useful to provide as an example of how to include things.